src/test/java/HW16.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HW16 extends BaseTest{
+//    Create a test case, registrationNavigation() using @Test annotation.
+//    Using Selenium, navigate to "https://bbb.testpro.io/".
+//    Click the Registration link.
+//    Verify that you are redirected to Registration page using Assert method.
+    @Test
+    public void registrationNavigation(){
+        WebDriver driver = new ChromeDriver();
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        Assert.assertEquals(driver.getCurrentUrl(), url);
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )
![image](https://user-images.githubusercontent.com/104761482/222364693-9f86bbb5-628a-4057-9c4a-a41b6ca54b44.png)